src/main/java/seedu/cafectrl/parser/ParserUtil.java

```diff
@@ -0,0 +1,16 @@
+package seedu.cafectrl.parser;
+
+import seedu.cafectrl.command.Command;
+import seedu.cafectrl.data.CurrentDate;
+import seedu.cafectrl.data.Menu;
+import seedu.cafectrl.data.Pantry;
+import seedu.cafectrl.data.Sales;
+import seedu.cafectrl.ui.Ui;
+
+/**
+ * Parser interface for external class to use Parser
+ */
+public interface ParserUtil {
+    Command parseCommand(Menu menu, String userInput, Ui ui,
+                                Pantry pantry, Sales sales, CurrentDate currentDate);
```

Comment:
Perhaps line 15 the space should be 8 spaces from line 14?
```suggestion
    Command parseCommand(Menu menu, String userInput, Ui ui,
            Pantry pantry, Sales sales, CurrentDate currentDate);
```